core/controllers/base.py

```diff
@@ -355,8 +355,10 @@ def validate_and_normalize_args(self):
             handler_class_name in
             handler_schema_constants.HANDLER_CLASS_NAMES_WITH_NO_SCHEMA
         ):
-            # TODO(#13155): Remove this clause once all the handlers have had
-            # schema validation implemented.
+            if self.URL_PATH_ARGS_SCHEMAS or self.HANDLER_ARGS_SCHEMAS:
+                raise self.InternalErrorException(
+                    'Remove handler from '
```

Comment:
The constant contains names of the handler so the message should be like -- "Remove the handler class name from ..."

Reply:
done